.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release builder
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release_build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-20.04]
+        php-versions: ['7.2', '7.3', '7.4', '8.0', '8.1']
+    steps:
+    - name: Install PHP ${{ matrix.php-versions }}
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+    - name: Checking PHP version
+      run: "php -v"
```

Comment:
Can be removed? PHP version should be shown by default.

Reply:
done!